### DISCUSSION TITLE
How to properly configure authorization servers and blocked servers

### DISCUSSION THREAD

**Discussion by bivashy** [U61vashka/MC-Auth-with-VK#103] [Q&A] [answered]
Source: https://github.com/bivashy/MC-Auth-with-Link/issues/102#issuecomment-1764738631

**Comment by MrAnrehah** [comment]
![image](https://github.com/bivashy/MC-Auth-with-Link/assets/32076073/bd1096a0-e858-40ac-afb3-12ab5e22ef59)
And what's wrong with the waterfall config itself?

**Comment by MrAnrehah** [comment]
And the removal of lobby eliminated the [error](https://github.com/bivashy/MC-Auth-with-Link/issues/102), but this error does not always manifest itself.
```
blocked-servers:
  - server
auth-servers:
  - auth
game-servers:
  - lobby
```

**Comment by bivashy** [comment, marked as answer]
I think you should change `force_default_server` field into `true`.

![изображение](https://github.com/bivashy/MC-Auth-with-Link/assets/85439143/2d20920b-bca0-407b-a454-608c00a977bc)

**Reply by MrAnrehah** [reply]
I want to inform you that sometimes when I log in to the server, I stay in auth, I have to re-log in.

**Reply by bivashy** [reply]
Try to replicate and provide more info. Because i cannot replicate it. Also it may be caused by other plugins.

**Reply by MrAnrehah** [reply]
![image](https://github.com/bivashy/MC-Auth-with-Link/assets/32076073/1f71e69d-707a-4b0e-980a-cc86fc22cdcb)
Also, if you don't log in for a while, then he writes. (Invalid move player packet received)
Well, also, if you remove the plugin, then everything works fine.
At velocity, the plugin works perfectly.

**Reply by bivashy** [reply]
mcAuth doesn't modify/affect packets in BungeeCord platform. `Invalid move player packet received` indicates client issues/improper configuration of server. 

Also this 'error' must be sent by backend server (not the proxy one), like Spigot/Paper/Purpur etc. You should provide as much info as you can. Because i cannot fix bug/error without replicating it.

**Reply by MrAnrehah** [reply]
To repeat the error, create 3 servers (authorization, lobby, survival) log in using the mcauth plugin, log in to survival, after that, turn off the server while on survival, then turn it on, then log in to the server, and you are more likely to receive `Invalid move player packet received`

**Reply by bivashy** [reply]
As i've said earlier this issue has nothing to do with mcAuth. It is issue with your backend server, see: https://github.com/PaperMC/Paper/issues/8222. mcAuth cannot interact with your backend servers directly.
